pkg/apis/config/defaults_test.go

```diff
@@ -212,7 +208,7 @@ func TestTemplating(t *testing.T) {
 				},
 			})
 			if err != nil {
-				t.Errorf("Error parsing defaults: %v", err)
+				t.Fatal("Error parsing defaults:", err)
```

Comment:
(nit) Missing trailing space before the error.
```suggestion
				t.Fatal("Error parsing defaults: ", err)
```

Reply:
Same as `Error`, `testing.T.Fatal` adds a space between the arguments because it uses `fmt.Sprintln` internally.

Reply:
My bad, discarding